app/src/main/java/com/socketmobile/stockcount/helper/CaptureHelper.kt

```diff
@@ -1,4 +1,6 @@
 /**  Copyright © 2018 Socket Mobile, Inc. */
 
 package com.socketmobile.stockcount.helper
-
+fun isSocketCamDevice(deviceName: String): Boolean {
+    return deviceName == "SocketCam C820" || deviceName == "SocketCam C860"
+}
```

Comment:
How about making this a DeviceClient extension? In any case it shouldn't be based on the device name

```suggestion
fun DeviceClient.isSocketCamDevice(): Boolean {
    return deviceType == kModelSocketCamC820
}
```